From aff6b63fbc9c0ae4ca9aab721ddbc9b4e1ca1cbc Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:07 -0400
Subject: [tools] perf: Get rid of a duplicate va_end() in error reporting routine

Message-id: <1408953210-26343-180-git-send-email-jolsa@redhat.com>
Patchwork-id: 88969
O-Subject: [PATCH RHEL7.1 BZ1133083 179/322] perf tools: Get rid of a duplicate va_end() in error reporting routine
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 73db8f82619b7538d9b4badfe13f3ab2fef7d9b3
Author: Namhyung Kim <namhyung.kim@lge.com>
Date: Thu Dec 19 16:00:08 2013 +0900

description
===========
The va_end() in _eprintf() should be removed since the caller also
invokes va_end().
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/debug.c b/tools/perf/util/debug.c
index 8640a91..299b555 100644
--- a/tools/perf/util/debug.c
+++ b/tools/perf/util/debug.c
@@ -25,7 +25,6 @@ static int _eprintf(int level, const char *fmt, va_list args)
    ui_helpline__vshow(fmt, args);
   else
    ret = vfprintf(stderr, fmt, args);
-  va_end(args);
  }
 
  return ret;
-- 
1.7.1